src/plugins/charts/bar/inspector/SeriesOptions.vue

```diff
@@ -21,7 +21,7 @@
 -->
 <template>
   <ul>
-    <li class="c-tree__item menus-to-left" :class="aliasCss">
+    <li class="c-tree__item menus-to-left" :class="aliasCss" role="treeitem">
```

Comment:
this probably will break some tests (for good reason), i think the only other thing that has treeitem role is the main tree. but this seems like the right role for this to have